back/src/app.ts

```diff
@@ -95,6 +89,16 @@ io.on('connection', function (socket: Socket) {
   });
 });
 
-const server = http.listen(3000, function () {
+const server = http.listen(3000, () => {
   console.log('listening on *:3000');
+  const database = envConstants.isApiMock ? 'mock' : 'mongo';
+  console.log(`database with ${database}`);
+  if (!envConstants.isApiMock && envConstants.mongoUrl) {
+    connect(envConstants.mongoUrl, {
+      useUnifiedTopology: true,
+      useNewUrlParser: true,
+    })
+      .then(() => console.log('Mongo database connected'))
+      .catch((err) => console.log('Mongo can not connect', err));
```

Comment:
I think we should move away this from the main app.ts, and place in another file, maybe a storage sibling or something like that? something like MongoConnection